app/controllers/idv/by_mail/request_letter_controller.rb

```diff
@@ -64,11 +64,7 @@ def confirm_profile_not_too_old
       end
 
       def step_indicator_current_step
```

Comment:
Could we remove this method and the `@step_indicator_current_step` variable and hardcode `:verify_address` in the view?